Gems/AtomLyIntegration/EditorModeFeedback/Code/Source/EditorModeFeedbackSystemComponent.h

```diff
@@ -40,8 +42,16 @@ namespace AZ
             // EditorModeFeedbackInterface overrides ...
             bool IsEnabled() const override;
 
-        private:
-            //! Settings registery override for enabling/disabling editor mode feedback.
+        private:            
+            ///////////////////////////////////////////////////////////////////////
+            // AzToolsFramework::EditorEntityContextNotificationBus overrides
```

Comment:
nit: `EditorEntityContextNotificationBus overrides ...` (omit namespace in this context for brevity and include ellipsis)